test/e2e/garbage_collector.go

```diff
@@ -107,13 +107,14 @@ func newOwnerRC(f *framework.Framework, name string, replicas int32) *v1.Replica
 }
 
 // verifyRemainingDeploymentsAndReplicaSets verifies if the number
-// of the remaining deployment and rs are deploymentNum and rsNum.
-// It returns error if the communication with the API server fails.
+// of the remaining deployments, replica set and pods are deploymentNum,
+// rsNum and podNum. It returns error if the communication with the API
+// server fails.
 func verifyRemainingDeploymentsAndReplicaSets(
```

Comment:
Sorry i didn't notice this earlier, we should update the function name. verifyRemainingDeploymentsReplicaSetsPods? Or maybe just verifyRemainingObjects.